config/v1/types_cluster_version.go

```diff
@@ -331,9 +335,14 @@ const (
 	// The following resources are taken into account:
 	// - deploymentconfigs
 	ClusterVersionCapabilityDeploymentConfig ClusterVersionCapability = "DeploymentConfig"
+
 	// ClusterVersionCapabilityImageRegistry manages the image registry which
 	// allows to distribute Docker images
 	ClusterVersionCapabilityImageRegistry ClusterVersionCapability = "ImageRegistry"
+
+	// ClusterVersionCapabilityOLM manages the Operator Lifecycle Manager (OLM)
+	// which itself manages the lifecycle of operators
+	ClusterVersionCapabilityOLM ClusterVersionCapability = "OLM"
```

Comment:
Nit: everyone on this pull request knows what OLM unpacks to, but to make life easier for new OpenShift users/customers, can we unpack this to `OperatorLifecycleManager`?

Reply:
I like that, changing to full name to remove ambiguity. Thanks!